spock-core/src/main/java/org/spockframework/mock/runtime/ByteBuddyMockMaker.java

```diff
@@ -0,0 +1,55 @@
+package org.spockframework.mock.runtime;
+
+import org.spockframework.mock.CannotCreateMockException;
+import org.spockframework.util.ReflectionUtil;
+import org.spockframework.util.ThreadSafe;
+import spock.mock.MockMakers;
+
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.Set;
+
+@ThreadSafe
+public class ByteBuddyMockMaker implements IMockMaker {
+  public static final String ID = MockMakers.byteBuddy.getMockMakerId();
```

Comment:
I think it is better, when the IDs Strings defined by Spock can be found in a single locations, because you would need these Strings for the `mockMaker.preferredMockMaker` configuration.
If you navigate through the code you would use `MockMakers` and there you would find the IDs.

Reply:
Alright, I don't have too strong feelings about this one, so let's keep it. :-)